lib/config.js

```diff
@@ -333,7 +333,11 @@ function Config(options) {
 
     if (useConfig) {
         debug("Using command line config " + useConfig);
-        this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
+        try {
+            this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
```

Comment:
>  work like `require`

We can easily switch to `require`-based behavior by updating [this line](https://github.com/eslint/eslint/blob/10ec3c1ab7d88ec9d58205a46c5a75d1db5b36de/lib/config.js#L336) like below:

```diff
- this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
+ this.useSpecificConfig = loadConfig(useConfig);
```

But, how does it work when the `--config` option takes a relative YAML path?

```
eslint --config ./path/to/config.yml .
```

`require` cannot directly parse YAML.